_grades/A10.md

```diff
@@ -0,0 +1,117 @@
+
+You make achivable promises and stick to your commitments
+
+## Impact 
+
+### Solution Ownership
+
+You focus on business results and higher profit margin, while organizing successful delivery
+
+You are able to drive projects and services from conception through to completion
+
+You create new processes and methods that can be implemented across multiple areas
+
+You proactively develop and implement efficiency improvements that can be implemented in a number of areas
+
+You resolve large and complex issues and in doing so strengthen the relationship with the client
+
+### Technical Assessment and Adoption
+
+You identify and assess business change situations and the technical solutions that can support a client organization, and define its governance processes and program management
+
+You can assess market trends, ensuring your team has all the modern skills required to deliver high quality products, and outperform competing companies
+
+### Business Awareness
+
+You demonstrate a passion for our business and constant hunger for outstanding performance
+
+You understand and analyze the complexities and opportunities inherent in a global business environment
+
+You understand the political infrastructure of board level situations and operate accordingly
+
+You create and drive business plans for your area
+
+You have completed the required finance and commercial courses for this grade
+
+### Developing Our Team
+
+You understand how we make money and how you and your team or decisions fit into that
+
+You lead your team to strive to drive profitable growth and exceed expectations 
+
+You implement effective recruitment strategies to build the organisation and meet business needs
+
+### Winning New Work
+
+You create opportunities to develop markets/clients and adapt our business
+
+You devise and set the business strategy to achive market strength and business growth
+
+You have advocated wins in the market, leveraging your high expertise and personal insights
+
+You constantly bring value to widen positions in a sustainable way and build long-term partnerships with clients
+
+You influence and develop the future direction of our client base
+
+## Team Leadership
+
+You are an engagement lead and a driving force in the partnership between Capgemini and the client
+
+You lead and motivate multi-functional teams in dispersed or complex environments
+
+You embody the values of Capgemini, and convey the company vision to your teams
+
+You suggest and lead new initiatives both within the team and the wider company
+
+You have substantial knowledge of the wider Group's capabilities, and encourage co-operative team working across the business, to break silos and leverage the best teams, expertise and assets
+
+You take a high degree of responsibility and ownership of people issues within your area
+
+You boost teams’ and individuals’ potential to improve their engagement, performance, and career experience
```

Comment:
I don't think you can boost someone's potential - you can boost their morale or their desire to reach their potential?

Reply:
Yes good point, reworded to say: You allow teams’ and individuals’ to fulfil their potential